src/app/components/elements/FormattedAsset.jsx

```diff
@@ -8,6 +8,11 @@ const FormattedAsset = ({ amount, asset, classname }) => {
     if (amount && typeof amount === 'string') {
         amount = parsePayoutAmount(amount);
     }
+
+    const minimumAmountForPayout = 0.02;
+    if (amount < minimumAmountForPayout) {
+      amount = 0.00;
```

Comment:
The posted screenshot shows a pending payout of $0.01 still showing as 1 cent, which seems to contradict the logic here. Can we remove the amount reset?

Reply:
ah yea, will do that